raiden/app.py

```diff
@@ -36,40 +36,41 @@
 
 class App:  # pylint: disable=too-few-public-methods
     DEFAULT_CONFIG = {
-        'host': '',
-        'port': INITIAL_PORT,
-        'external_ip': '',
-        'external_port': INITIAL_PORT,
         'privatekey_hex': '',
         'reveal_timeout': DEFAULT_REVEAL_TIMEOUT,
         'settle_timeout': DEFAULT_SETTLE_TIMEOUT,
         'database_path': '',
-        'msg_timeout': 100.0,
+        'transport_type': 'udp',
         'transport': {
-            'retry_interval': DEFAULT_TRANSPORT_RETRY_INTERVAL,
-            'retries_before_backoff': DEFAULT_TRANSPORT_RETRIES_BEFORE_BACKOFF,
-            'throttle_capacity': DEFAULT_TRANSPORT_THROTTLE_CAPACITY,
-            'throttle_fill_rate': DEFAULT_TRANSPORT_THROTTLE_FILL_RATE,
-            'nat_invitation_timeout': DEFAULT_NAT_INVITATION_TIMEOUT,
-            'nat_keepalive_retries': DEFAULT_NAT_KEEPALIVE_RETRIES,
-            'nat_keepalive_timeout': DEFAULT_NAT_KEEPALIVE_TIMEOUT,
+            'udp': {
+                'external_ip': '',
+                'external_port': INITIAL_PORT,
+                'host': '',
+                'nat_invitation_timeout': DEFAULT_NAT_INVITATION_TIMEOUT,
+                'nat_keepalive_retries': DEFAULT_NAT_KEEPALIVE_RETRIES,
+                'nat_keepalive_timeout': DEFAULT_NAT_KEEPALIVE_TIMEOUT,
+                'port': INITIAL_PORT,
+                'retries_before_backoff': DEFAULT_TRANSPORT_RETRIES_BEFORE_BACKOFF,
+                'retry_interval': DEFAULT_TRANSPORT_RETRY_INTERVAL,
```

Comment:
Can you, please, add respective values for `retry_interval` and `retries_before_backoff` to the `matrix` key, as default times 5 or so, so we can have it configured here and remove [this ugly hack](https://github.com/raiden-network/raiden/blob/45e017ee38915c6b91c35edf37c18afeb898aa3f/raiden/ui/cli.py#L708-L709) ?

Reply:
Do you want new settings for this?

Reply:
Not _new_, the same, but on/for `matrix`, and [by default the current ones] multiplied by 5.